src/getPath.unit.js

```diff
@@ -0,0 +1,63 @@
+const test = require('ava');
+const path = require('path');
+const getPath = require('./getPath');
+
+test('throws if data is missing', (t) => {
+    t.throws(() => getPath({}), /base property/);
+    t.throws(() => getPath({ base: '' }), /source property missing on paths/);
+    t.throws(() => getPath({ base: '', source: '' }, {}), /source property missing on type config/);
+});
+
+test('correctly resolves relative path', (t) => {
+    const resolved = getPath(
+        {
+            base: '../test',
+            source: 'src',
+        },
+        {
+            source: 'js',
+        },
+    );
+    t.is(resolved, path.join(process.cwd(), '../test/src/js'));
+});
+
+test('correctly resolves absolute base path', (t) => {
+    const resolved = getPath(
+        {
+            base: process.cwd(),
```

Comment:
Thanks! Used pure strings instead. This is what you intended to say, right?

Reply:
I wanted to say that you should not use the same path your script will use by default. It's not a question of being a string, it's just important that the `base` path is different from your fallback (otherwise you cannot be sure if your path building helper does respect the base path you passed, even though it might be obvious).

Reply:
But you fixed it correctly ;)